waku/v2/peermanager/peer_manager.go

```diff
@@ -120,3 +109,86 @@ func (pm *PeerManager) pruneInRelayConns() {
 		}
 	}
 }
+
+// AddPeer adds peer to the peerStore and also to service slots
+func (pm *PeerManager) AddPeer(address ma.Multiaddr, origin wps.Origin, protocols ...protocol.ID) (peer.ID, error) {
+	info, err := peer.AddrInfoFromP2pAddr(address)
+	if err != nil {
+		return "", err
+	}
+
+	//Add Service peers to serviceSlots.
+	for _, service := range protocols {
+		if service != WakuRelayIDv200 {
+			pm.AddPeerToServiceSlot(service, info.ID)
+		}
```

Comment:
Since AddPeerToServiceSlot already verifies that the protocol isnt WakuRelay, maybe we can replace this for:
```suggestion
		pm.AddPeerToServiceSlot(service, info.ID)
```